buzzmobile/tests/sense/test_inputer.py

```diff
@@ -1,55 +1,21 @@
-import rostest
 import unittest
-import rospy
-import time
 from std_msgs.msg import String
-import roslaunch
+from tests import rostest_utils
 
-PKG = 'buzzmobile'
 NAME = 'test_inputer'
 
 
+@rostest_utils.with_roscore
 class TestInputer(unittest.TestCase):
 
-    def run_node_with_callback(self, callback):
-        rospy.init_node(NAME, anonymous=True)
-
-        rospy.Subscriber('destination', String, callback)
-        timeout = time.time() + 2.0
-
-        while time.time() < timeout and not rospy.is_shutdown():
-            time.sleep(0.1)
-
-    def __init__(self, *args):
-        super().__init__(*args)
-
-
     def setUp(self):
         self.success = False
-        node = roslaunch.core.Node('buzzmobile', 'inputer.py')
-        launch = roslaunch.scriptapi.ROSLaunch()
-        launch.start()
-        self.process = launch.launch(node)
-
-    def test_inputer_no_input(self):
-        def cb(data):
-            self.success = (data.data == '')
-
-        self.run_node_with_callback(cb)
-        assert self.success
 
+    @rostest_utils.launch_node('buzzmobile', 'inputer.py', 'params.launch')
     def test_sanity(self):
         def cb(data):
             self.success = (data.data == b'success')
 
-        self.run_node_with_callback(cb)
-        assert not self.success
-
-    def tearDown(self):
-        self.process.stop()
-
-
-if __name__ == '__main__':
-    rostest.rosrun(PKG, NAME, TestInputer)
-
+        with rostest_utils.test_node('direction', String, cb):
```

Comment:
So `test_node` is way for us to check on a message that is maybe being published by the node launched in `launch_node`, correct? And if `launch_node` depends on anything, we can use a `mock_node`.

I know `test_node` creates a node that subscribes to `launch_node`'s message in the background, but it's kind of confusing to me to see a test that has:

```python
test_node('message_name')
# instead of...
test_node('node_name')
```

What do you think of renaming `test_node` to something like `check_topic`, or `check_message`, etc... I don't really know what the best name would be..